cg/knex-config-factory.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function() {
+  const dbName = global.dbName || process.env.DB_DATABASE;
```

Comment:
this is a hack, so that we can inject dbName in tests – I'm open to better solutions